WeatherForecast/user_service/templates/registration/password_reset_complete.html

```diff
@@ -0,0 +1,17 @@
+{% extends "admin/base_site.html" %}
+{% load i18n %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
+&rsaquo; {% translate 'Password reset' %}
+</div>
+{% endblock %}
+
+{% block content %}
+
+<p>{% translate "Your password has been set.  You may go ahead and log in now." %}</p>
```

Comment:
Tagi translate są używane do i18n, czyli internacjonalizacji, to trzeba użyć wszędzie jeśli już chcemy używać, w sensie tam, gdzie user dostaje tekst
https://docs.djangoproject.com/en/4.0/topics/i18n/translation/